scripts/exregional_run_analysis.sh

```diff
@@ -297,22 +301,23 @@ fi
 #           radar_tten converting code.
 #-----------------------------------------------------------------------
 
-cp_vrfy ${fixgriddir}/fv3_akbk                     fv3_akbk
-cp_vrfy ${fixgriddir}/fv3_grid_spec                fv3_grid_spec
+ln_vrfy -snf ${fixgriddir}/fv3_akbk                     fv3_akbk
+ln_vrfy -snf ${fixgriddir}/fv3_grid_spec                fv3_grid_spec
 
 if [ ${BKTYPE} -eq 1 ]; then  # cold start uses background from INPUT
-  cp_vrfy ${bkpath}/gfs_data.tile7.halo0.nc        gfs_data.tile7.halo0.nc_b
-  ncks -A -v  phis ${fixgriddir}/phis.nc           gfs_data.tile7.halo0.nc_b
+  ln_vrfy -snf ${bkpath}/gfs_data.tile7.halo0.nc        gfs_data.tile7.halo0.nc_b
+  ln_vrfy -snf ${fixgriddir}/phis.nc                    phis.nc
+  ncks -A -v  phis               phis.nc           gfs_data.tile7.halo0.nc_b
 
-  cp_vrfy ${bkpath}/sfc_data.tile7.halo0.nc        fv3_sfcdata
-  cp_vrfy gfs_data.tile7.halo0.nc_b                fv3_dynvars
+  ln_vrfy -snf ${bkpath}/sfc_data.tile7.halo0.nc        fv3_sfcdata
+  ln_vrfy -snf gfs_data.tile7.halo0.nc_b                fv3_dynvars
   ln_vrfy -s fv3_dynvars                           fv3_tracer
 
   fv3lam_bg_type=1
 else                          # cycle uses background from restart
-  cp_vrfy  ${bkpath}/fv_core.res.tile1.nc             fv3_dynvars
-  cp_vrfy  ${bkpath}/fv_tracer.res.tile1.nc           fv3_tracer
-  cp_vrfy  ${bkpath}/sfc_data.nc                      fv3_sfcdata
+  ln_vrfy  -snf ${bkpath}/fv_core.res.tile1.nc             fv3_dynvars
+  ln_vrfy  -snf ${bkpath}/fv_tracer.res.tile1.nc           fv3_tracer
+  ln_vrfy  -snf ${bkpath}/sfc_data.nc                      fv3_sfcdata
```

Comment:
According to the "NCO implementation standards" document, symbolic links to resources outside of the application directory or package (e.g. links to absolute paths) are not allowed. So links such as these may need to be removed before the package is delivered to NCO.

Reply:
@AnnetteGibbs-NOAA Thanks!  Yes, current change to "ln_vrfy" is to speed up run time. It will be modified when we are ready to deliver it to NCO.